### PR TITLE
Update free-courses-en.md with Learn Blockchain & Solidity Development Full Course 2024

### PR DESCRIPTION
**Pull request by anurags10**
## What does this PR do?
Add resource 

### Description
Learn Blockchain & Solidity Development Full Course 2024

### Why is this valuable (or not)?
This course covers blockchain and Solidity development, up to date with multiple projects for hands-on learning.
### How do we know it's really free?
It's on youtube.

### For book lists, is it a book? For course lists, is it a course? etc.
course

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -330,6 +330,7 @@
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
 * [Ethereum Developer Bootcamp](https://www.alchemy.com/university/courses/ethereum) - Alchemy University
 * [Learn Blockchain](https://www.youtube.com/playlist?list=PLlp912GlUiC1xPnwVmKgIiuMERetMwre9) - Roomyan
+* [Learn Blockchain & Solidity Development Full Course 2024](https://youtu.be/jcgfQEbptdo?feature=shared) - Dapp University
```
Comment (eshellman) [inline courses/free-courses-en.md:333]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
NO SHORTENED LINKS

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -330,6 +330,7 @@
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
 * [Ethereum Developer Bootcamp](https://www.alchemy.com/university/courses/ethereum) - Alchemy University
 * [Learn Blockchain](https://www.youtube.com/playlist?list=PLlp912GlUiC1xPnwVmKgIiuMERetMwre9) - Roomyan
+* [Learn Blockchain & Solidity Development Full Course 2024](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUNcUVXtn2KXiE1Ui9B5UrY) - Gregory (Dapp University)
```
Comment (eshellman) [inline courses/free-courses-en.md:333]: Don't invent titles! Use the title given by the resource.
for Youtube, use the channel name - "Dapp University", not "Gregory"

Comment (anurags10) [reply courses/free-courses-en.md]: @eshellman fixed.

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -329,6 +329,7 @@
 * [Build and Deploy Your First Decentralized App with Etherem](https://www.udemy.com/course/your-first-decentralized-app/) - Gary Simon (Udemy)
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
 * [Ethereum Developer Bootcamp](https://www.alchemy.com/university/courses/ethereum) - Alchemy University
+* [Free Blockchain Development Courses](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUNcUVXtn2KXiE1Ui9B5UrY) - Dapp University (Gregory)
```
Comment (eshellman) [inline courses/free-courses-en.md:332]:
```suggestion
* [Free Blockchain Development Courses](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUNcUVXtn2KXiE1Ui9B5UrY) - Dapp University
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**